example/README.md

```diff
@@ -0,0 +1,13 @@
+React Ladda Button Example
+==============================
+
+This is a runnable demo showing how to use `react-ladda`
+
+## Running Example
+
+**In the example directory, run:**
+```
+$ npm install
+$ npm start
+```
+**Open [http://localhost:3000](http://localhost:3000) to view it in the browser.**
```

Comment:
I think we can remove this since `npm start` will automatically open the browser.